README.md

```diff
@@ -135,6 +149,48 @@ Show the contents of the blocked ips list:
 bundle exec catadog | jq 'select(.kind=="rc") | .response.body.target_files | .[] | select(.path | test("ASM_DATA/blocked_ips"))'
 ```
 
+## Docker Compose
```

Comment:
Is this section helpful to have? If so, is it clear and comprehensive?

Reply:
Nit: > Using Docker Compose

Reply:
Why cloning with `git clone https://github.com/DataDog/catadog` when we have `ghcr.io/datadog/catadog`?

I would expect to clone the repo only if you're doing active development, compilation. If you are a user who just wants to use the latest `catadog` in your own app's `docker-compose.yml`, I would expect this would describe a `catadog` service and any relevant additional config (such as the API key as you described.)

Reply:
Makes sense! I was originally thinking the use-case would be active development-related, but it makes more sense that the user would just want to incorporate `catadog` into their app's `docker-compose.yml`.